src/compiler/types.ts

```diff
@@ -1681,6 +1682,12 @@ module ts {
         UMD = 3,
     }
 
+    export const enum NewLineKind {
+        DEFAULT = 0,

```

Comment:
Since each of the options are optional properties, I don't think we need a default.